crypto/ocsp/ocsp_vfy.c

```diff
@@ -204,8 +205,13 @@ static X509 *ocsp_find_signer_sk(STACK_OF(X509) *certs, OCSP_RESPID *id)
     /* Calculate hash of each key and compare */
     for (i = 0; i < sk_X509_num(certs); i++) {
         x = sk_X509_value(certs, i);
-        if (!X509_pubkey_digest(x, EVP_sha1(), tmphash, NULL))
+        if ((md = EVP_MD_fetch(x->libctx, SN_sha1, x->propq)) == NULL)
             break;
+        if (!X509_pubkey_digest(x, md, tmphash, NULL)) {
```

Comment:
sometimes I do the following for these:

````
rv = X509_pubkey_digest(....);
EVP_MD_free(md);
if (!rv)
   break;
````

Reply:
Yeah, thought about it.  Didn't bother.

Reply:
Okay, did bother.